ui/src/components/common/SlidingSidebar/index.tsx

```diff
@@ -119,22 +144,24 @@ export function SlidingSidebar({
         height: "100%",
       }}
     >
-      <Box
-        sx={{
-          display: "flex",
-          flexDirection: "column",
-          justifyContent: "center",
-          height: "100vh",
-        }}
-      >
-        <img
-          onMouseDown={dragHandler}
-          src={slider}
-          alt="slider"
-          className={"sidebar-drag-icon"}
-          draggable={false}
-        />
-      </Box>
+      {!errorsProps && (
```

Comment:
Can we add "slide" boolean prop to top level SidebarProps w/ a default of true for this? 

Then when you call setSidebarProps, just add "slide: false". Just like type in the unnested props.

This will make any other future cases where slider isn't needed much easier.

Reply:
done